### DISCUSSION TITLE
Modder analytics

### DISCUSSION THREAD

**Discussion by Minenash** [modrinth/.github#2] [Feature requests]
It'd be really cool if you could get mod stats like how many downloads/views/points(for dev revenue) per day. Cursedforge had this until a couple months ago. I'm not asking for some fancy gui, but a downloadable spread sheet with the columns: date, views, downloads, and possible revenue points.

I'd assume it would just be stored in a database, then exported upon dev request. The biggest problem I can think of is storage requirements need for all the rows, since it would be over 600 rows a day, unless there's a more compact way to store it I'm not thinking about.

**Comment by triphora** [comment]
Closing since this has been implemented.
